### DISCUSSION TITLE
Context-Aware RecSys

### DISCUSSION THREAD

**Discussion by ElifEceErdem** [RUCAIBox/RecBole#1818] [Q&A] [answered]
I want to develop a recommendation engine using the RecBole library in Python.  The characteristic of my data is as follows:
- I have customers who purchase the hotels, so the customers are the users and the hotels are the items to be recommended. Users and hotels have their own features. I want to use these features in my model. I don't have any ratings or reviews of users. That is why, I will recommend hotels to users implicitly.
- Of course, the model should use the interaction between the user and the item as well as their features.
- I tried to use context-aware models, which request .inter, .user and, .item atomic files. But as I don't have any label (rating or review) I couldn't run it properly. (I got KeyError: 'label')
Can you recommend me, models that I can use in the RecBole library, which uses user and item features and the interaction between user and item, please?

**Comment by zhengbw0324** [comment, marked as answer]
We suggest that you perform negative sampling on the data before using the context-aware model.

**Reply by ElifEceErdem** [reply]
thanks a lot

**Reply by vinod724-envestnet** [reply]
@zhengbw0324  Are you suggesting that .inter files must have negative interactions in them and have 'label' or 'rating'?
